### DISCUSSION TITLE
Run time error with fooocus colab

### DISCUSSION THREAD

**Discussion by deep1996arch** [lllyasviel/Fooocus#3203] [Q&A] [answered]
run time error with fooocus colab, even I reinstall all driver but same are detected on my both device.RuntimeError: Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver

**Comment by mashb1t** [comment, marked as answer]
You're using an instance without GPU, which means you've selected it by accident or you've ran out of GPU tokens/minutes. Please check both and consider upgrading to Colab Pro for extended use.
